test/functional/basic/calendar-e2e-specs.js

```diff
@@ -0,0 +1,38 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { TESTAPP_CAPS } from '../desired';
+import { initSession, deleteSession, MOCHA_TIMEOUT } from '../helpers/session';
+import _ from 'lodash';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+if (!process.env.REAL_DEVICE) {
+
+  describe('XCUITestDriver - calendar', function () {
+    this.timeout(MOCHA_TIMEOUT);
+
+    let driver;
+
+    after(async () => {
+      await deleteSession();
+    });
+
+    it('should authorize calendar access if calendarAccessAuthorized == true', async () => {
+      driver = await initSession(_.defaults(TESTAPP_CAPS, {calendarAccessAuthorized: true}));
+      let checkCalendarButton = await driver.elementByName('Check calendar authorized');
+      await checkCalendarButton.click();
+      let authorizedStaticText = await driver.elementByName('authorized');
```

Comment:
you could just rewrite this as:

```js
await driver.elementByName('authorized');
```
(no need to check objectness of the return value)

since it will throw if it can't find the element